### ISSUE TITLE
Error: Unable to resolve "./util.inspect" in `swell.umd.min.js`

### ISSUE THREAD

**Issue by robwalkerco** [swellstores/swell-js#172]
After updating `swell-js` from `4.0.0` to `4.0.1` we are getting the following error when using `swell-js` in React Native:

`Unable to resolve "./util.inspect" from "node_modules/swell-js/dist/swell.umd.min.js"`

 This is caused by the following code in `node_modules/swell-js/dist/swell.umd.min.js`
 ``` js
...
var utilInspect = require('./util.inspect');
var inspectCustom = utilInspect.custom;
var inspectSymbol = isSymbol$1(inspectCustom) ? inspectCustom : null;
...
```

I tried to see what change between those versions to cause this issue, but its not obvious to me, so hopefully you may know!

**Comment by awwit** [comment]
@robwalkerco thanks for the bug report! This was fixed in `v4.1.0`.

**Comment by robwalkerco** [comment]
Thanks for fixing
